### ISSUE TITLE
Does SmolVLA version 0.4.4 seem to perform better than version 0.5.2?

### ISSUE THREAD

**Issue by captainfffsama** [huggingface/lerobot#3916]
### Ticket Type

❓ Technical Question

### Environment & System Info

```Shell

```

### Description

It seems that SmolVLA v0.4.4 performs better than SmolVLA v0.5.2. I used the same data and the same training settings, but the final loss of SmolVLA v0.4.4 was about one-third lower than that of v0.5.2. 

Does anyone know why this might happen? 

I’m using my own robot and collected around 200 demonstrations for grasping boxes.

<img width="2088" height="831" alt="Image" src="https://github.com/user-attachments/assets/d72a3a4c-741e-4c98-a588-ed83b7ad0737" />

blue line is 0.5.2 and orange line is 0.4.4

### Relevant logs or stack trace

```Shell

```

### Checklist

- [ ] I have searched existing tickets to ensure this isn't a duplicate.
- [ ] I am using the latest version of the `main` branch.
- [ ] I have verified this is not an environment-specific problem.

**Comment by Maximellerbach** [comment]
I think it's related to: https://github.com/huggingface/lerobot/pull/3434
the way the loss is computed has been changed to excluded padded actions
